src/features/instantiate/containers/InstanceSelection.tsx

```diff
@@ -43,15 +43,19 @@ function getDateDisplayString(timeStamp: string) {
 export function InstanceSelection({ instances, onNewInstance }: IInstanceSelectionProps) {
   const { data: applicationMetadata } = useApplicationMetadataQuery();
   const instanceSelectionOptions = applicationMetadata?.onEntry?.instanceSelection;
+  const selectedIndex = instanceSelectionOptions?.defaultSelectedOption;
   const { lang, langAsString, language } = useLanguage();
   const mobileView = useIsMobileOrTablet();
   const rowsPerPageOptions = instanceSelectionOptions?.rowsPerPageOptions ?? [10, 25, 50];
-  const defaultSelectedOption = instanceSelectionOptions?.defaultSelectedOption ?? 0;
 
+  const doesIndexExist = (selectedIndex: number) =>
+    rowsPerPageOptions.length - 1 >= selectedIndex && selectedIndex >= 0 ? selectedIndex : 0;
```

Comment:
If this is not used anywhere else you could just have it as a boolean instead of a function returning a boolean